lib/update_overlay.ex

```diff
@@ -0,0 +1,18 @@
+defmodule Membrane.OverlayFilter.UpdateOverlay do
+  @moduledoc """
+  Specifies the overlay update notification.
+  """
+
+  @enforce_keys [:overlay, :x, :y, :blend_mode]
+  defstruct @enforce_keys
+
+  @typedoc """
+  Specifies the overlay update.
+  """
+  @type t :: %__MODULE__{
+          overlay: Path.t() | Vix.Vips.Image.t(),
+          x: integer() | :center | :left | :right,
+          y: integer() | :middle | :top | :bottom,
+          blend_mode: Image.BlendMode.t()
```

Comment:
I believe we could unify the typespecs for the filter options and for the update overlay struct.